py/server/deephaven/table.py

```diff
@@ -319,6 +320,34 @@ def meta_table(self) -> Table:
     def j_object(self) -> jpy.JType:
         return self.j_table
 
+    def get_table_attributes(self) -> Dict[str, Any]:
```

Comment:
Do we want to offer the calls that allow users to fetch a single attribute, or subset of attribtues?

Reply:
Unless we have or going to have a huge number of such attributes, I don't think it is necessary.